### ISSUE TITLE
Deprecated download link

### ISSUE THREAD

**Issue by silas229** [PatrickRorth/Twitch.TV#24]
The Link to the Dropbox is deprecated

**Comment by rwnicholas** [comment]
I'm not the project owner, but maybe I can help.
I believe that the old method for installing apps is deprecated, and you should install by this [tutorial](http://webostv.developer.lge.com/develop/app-test/#preparingAccount)
I built a version with a tweak that is under request. [Download here](https://github.com/MarechalLima/Twitch.WebOS/raw/master/com.marechallima.twitchtv_0.0.6_all.ipk)

**Comment by danilokleber** [comment]
Hi @MarechalLima I'm also from Brazil. Can you help me on how to install it? I've done everything up to enabling Developer Mode. But I don't how to actually install the app. I'm using a Mac. Thanks in advance!

**Comment by gabscortes** [comment]
I have like 1 year streaming Twitch and other websites on my WebOS 2.0 through TV Cast app.

**Comment by danilokleber** [comment]
How do you do that @gabscortes?

**Comment by gabscortes** [comment]
https://www.reddit.com/r/Twitch/comments/5f5dlw/how_to_watch_twitch_on_lg_webos_tvs/

**Comment by nitrnitr** [comment]
@gabscortes is tv cast working for you now? Used for a year or so and it isn't working since last month. Stream is no longer starting for me and they don't reply my emails. Also last time I checked this app it didn't work either on webos 4.3 -_-

**Comment by gabscortes** [comment]
Yes it's still working for me, but my TV can only play 720p30fps on WiFi (I think that my TV's Wi-Fi is broken because I can watch 1080p60 smoothly if I connect the Ethernet). Smart TVs are not worth for watching twitch. Buy a Xiaomi Mi Box TV or a Chromecast for better experience (or just connect a PC with HDMI =).

**Comment by mauriciabad** [comment]
Not working anymore:
https://www.dropbox.com/s/7np16b29jye3lfk/TwitchTV.zip?dl=0

Maybe it'd be a good idea to use Git tags for releases if it's possible :)

**Comment by mordred2222** [comment]
I'm working on a new version with remote control working... 
Stable version will be posted on a dedicated website...

**Comment by Tizana** [comment]
> I'm working on a new version with remote control working...
> Stable version will be posted on a dedicated website...

so i'm hesitating between buying a new LG smart TV because of twitch !!!! is your app that you published works fine on Webos ?

**Comment by CharlesMagnuson** [comment]
> I'm working on a new version with remote control working...
> Stable version will be posted on a dedicated website...

@mordred2222 Is there any chance you've published your work yet?

**Comment by mordred2222** [comment]
Published there...

https://leweb.be/twitch-on-webos/




Le mer. 1 janv. 2020 à 03:13, Charles Magnuson <notifications@github.com> a
écrit :

**Comment by Deph0** [comment]
> Published there... https://leweb.be/twitch-on-webos/

Hi, looks like you've removed the download due to twitch publishing their own version...
however, i'm running an older version which has webos 3.4 and offical twitch app requires webos 3.5 which i cant update to. 
Do you mind uploading it to github releases or similar ?
